Tests/test_GenomeDiagram.py

```diff
@@ -70,21 +64,20 @@ def fill_and_border(base_color, alpha=0.5):
 
 
 def apply_to_window(sequence, window_size, function, step=None):
-    """ apply_to_window(sequence, window_size, function) -> [(int, float),(int, float),...]
-
-        o sequence      Bio.Seq.Seq object
-
-        o window_size   Int describing the length of sequence to consider
+    """Returns a list of (position, value) tuples for fragments of the passed
```

Comment:
Minor: Try to start the docstring with a single line description then a blank line as per PEP257 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings